docs/meshstack.aws.md

```diff
@@ -0,0 +1,8 @@
+---
+id: meshstack.aws
+title: Amazon Web Services
+---
+
+*AWS* is a proprietary public cloud platform provided by Amazon Web Services. Meshstack supports project and user management for AWS to include AWS services into cloud projects managed by Meshstack.
```

Comment:
d: As AWS stands for Amazaon Web Services, should we better just say ' provided by Amazon'?

Reply:
It's a separate entity: 
![image](https://user-images.githubusercontent.com/24991455/45417113-b516b200-b681-11e8-9d1c-bbe70666c189.png) so it is fair to say Amazon Web Services, Inc provides the AWS platform.